cli/src/main/java/org/jboss/as/cli/impl/aesh/SynopsisGenerator.java

```diff
@@ -24,6 +24,7 @@
 import java.util.Objects;
 import java.util.ResourceBundle;
 import java.util.Set;
+import java.util.LinkedHashSet;
```

Comment:
Instead of adding this import here, can you please keep the list of imports sorted by adding this line between Line 21 and 22?